detection_rules/devtools.py

```diff
@@ -721,6 +733,11 @@ def add_github_meta(this_rule: TOMLRule, status: str, original_rule_id: Optional
 @click.pass_context
 def deprecate_rule(ctx: click.Context, rule_file: Path):
     """Deprecate a rule."""
+    if RULES_CONFIG.bypass_version_lock:
+        click.echo('Cannot deprecate a rule when the versioning strategy is configured to bypass the version '
```

Comment:
Can we invert this message to basically say this is only _needed_ when using the lock strategy - otherwise all they need to do is update the maturity (and move or track it however they want)

Reply:
delete. This is an internal command.